### ISSUE TITLE
@skpm/extract-loader bad path

### ISSUE THREAD

**Issue by liqueflies** [skpm/with-webview#20]
Hello,
I'm dealing with an issue that appears when I try to load an assets on the web view.

If I use an image the result will be:  src="de4ae82677652bf9dd11a5f58bc47826.png";"

the `";` in addition does not break anything in html, but in css yes.

Trying to inspect I've found these two lines on extractLoader.js

```
    if (src.split("_webpack_resources/")[1]) {
        return src.split("_webpack_resources/")[1].split('").path()')[0];
    }
```

I don't know what those splits does but they breaks the filename.

Any ideas?

**Comment by greatpixels** [comment]
Hi, 
I have same issue.
Also, font face has same semi column problem `";` in css
```
@font-face {
  font-family: 'Icons';
  src: url(8e3c7f5520f5ae906c6cf6d7f3ddcd19.eot";);
  src: url(0ab54153eeeca0ce03978cc463b257f7.woff2";) format('woff2'), 
  url(faff92145777a3cbaf8e7367b4807987.woff";) format('woff'), 
  url(b87b9ba532ace76ae9f6edfe9f72ded2.ttf";) format('truetype');
}
```

**Comment by emptykid** [comment]
I face the same issue when adding font-awesome to my page
so I just fork the repository and fixed the bug by hand

```
if (src.split("_webpack_resources/")[1]) {

        return src.split("_webpack_resources/")[1].split('").path()')[0].replace(/";/g, '');

 }
```

remove the "@skpm/extract-loader": ^2.0.2
install npm package from github 
`
npm install --save-dev github:emptykid/extract-loader
`
everything works fine now.

**Comment by mathieudutour** [comment]
do you want to send a PR instead of maintaining a fork?

**Comment by mathieudutour** [comment]
I fixed it in `@skpm/extract-loader@2.0.3`
